misc/system.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# Base configuration file for DAQ.
+#
+# This setup will create and test one faux device. To specify a real USB
+# adapter, modify the intf_names entry with the appripriate interface name(s).
+# The startup_cmds line can then be commented out to prevent the (unnecessary)
+# startup of the faux test device.
+
+# Description description for dashboard.
```

Comment:
this duplicated description is still there

Reply:
updated